homeassistant/components/bayesian/binary_sensor.py

```diff
@@ -185,10 +187,6 @@ def async_threshold_sensor_state_listener(event):
             When a state changes, we must update our list of current observations,
             then calculate the new probability.
             """
-            new_state = event.data.get("new_state")
-
-            if new_state is None or new_state.state == STATE_UNKNOWN:
-                return
```

Comment:
Why remove these lines ? they usually address instability of sources/templates during HA boot up

Reply:
[f004247](https://github.com/home-assistant/core/pull/67631/commits/f0042474c8ed7f2b396d258757473d250ea95412)